app.js

```diff
@@ -764,6 +767,7 @@ async function main() {
             staticHeaders(res);
             res.end('User-agent: *\nSitemap: https://godbolt.org/sitemap.xml\nDisallow:');
         })
+        .use('/template_screenshots', express.static('etc/template_screenshots/'))
```

Comment:
Is there a better way to serve static assets?

Reply:
This is good for local users, I think. We'd probably upload the templates to our CDN on S3, so there probably needs to be a setting that swaps the URL to `https://static.ce-cdn.net/template_screenshots/...`

Reply:
What's needed to deploy the images to the cdn?

Reply:
It's complicated. Our static files are zipped up and uploaded to S3 in our github workflow. (here https://github.com/compiler-explorer/compiler-explorer/blob/main/.github/workflows/test-and-deploy.yml#L76)

But then only once we decide to deploy the version, the static files will be uploaded to the CDN using this function: https://github.com/compiler-explorer/infra/blob/33265d8edd518a31d96cbf36ee3f4345204f24ef/bin/lib/cli/builds.py#L49

Reply:
Thanks, I've updated how images are handled. Now using the same webpack magic used for the language icons 🤞

Reply:
Didn't realize that was possible, well done